sdc/tests/tests_perf/generator.py

```diff
@@ -0,0 +1,104 @@
+import time
+from sdc.io.csv_ext import to_varname
+from typing import NamedTuple
+
+
+class TestCase(NamedTuple):
+    name: str
+    params: str
+    size: int = None
+    call_expression: str = None
+
+
+def gen(cases, method, class_add, type, prefix=''):
+    for params in cases:
+        test_params = TestCase(*params)
+        name = test_params.name
+        if test_params.params:
+            name += "_" + to_varname(test_params.params)
```

Comment:
Can `test_params.params` consist of unavailable symbols or start with digit? Could you provide an example? If not using of `to_varname` is excess. I think the one who add new test case is responsible for data correctness.

Reply:
The name of the test depends on the parameters that were passed to it so that there are no matching tests.

Reply:
I see that the name depends on parameters but my question was "is a call of `to_varname` exactly required?". If you will do so:
`name += f'{name}_{test_params.params}'`, will it break at least some test?

Reply:
to_varname need, else name of test will be "test_drop_columns="f0" ", for example.